src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/DwpUploadResponseHandler.java

```diff
@@ -115,14 +117,19 @@ private void handleUc(Callback<SscsCaseData> callback) {
                 description = "update to response received event as there is a dispute.";
             }
 
-            SscsCaseData caseData = callback.getCaseDetails().getCaseData();
-
             caseData.setDwpState(DwpState.RESPONSE_SUBMITTED_DWP.getId());
 
             ccdService.updateCase(caseData, callback.getCaseDetails().getId(),
                 EventType.DWP_RESPOND.getCcdType(), "Response received",
                 description, idamService.getIdamTokens());
         }
+
+        if (caseData.getJointParty() != null && StringUtils.equalsIgnoreCase("Yes", caseData.getJointParty())) {
+            ccdService.updateCase(caseData, callback.getCaseDetails().getId(),
+                EventType.JOINT_PARTY_ADDED.getCcdType(), "Joint party added",
+                "description", idamService.getIdamTokens());
```

Comment:
Can you give the description a better sentence than just the word "description"?